extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/handler/TradfriGatewayHandler.java

```diff
@@ -79,17 +79,40 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void initialize() {
         GatewayConfig configuration = getConfigAs(GatewayConfig.class);
+
+        if (isConfigurationValid(configuration)) {
+            scheduler.schedule(new Runnable() {
```

Comment:
Could you use Java8 syntax here? (`schedule(() -> {... }, 0, TimeUnit.SECONDS);`)

Reply:
Will do it.